pkg/testsuites/standard_suites.go

```diff
@@ -55,6 +55,7 @@ var staticSuites = []ginkgo.TestSuite{
 		},
 		Parallelism:          30,
 		MaximumAllowedFlakes: 15,
+		TestTimeout:          20 * time.Minute,
```

Comment:
I think you need to have a discussion with the maintainers regarding this - did it occur on another comms channel?

Reply:
moved this test to ipsec test suite, so timeout change is not needed here.